tests/golden_path.spec.ts

```diff
@@ -1,7 +1,10 @@
-import { test, expect, Page  } from '@playwright/test';
+import { test, expect, Page,  } from '@playwright/test';
 
-test.describe.serial('Golden path', () => {
+const APP_URL = process.env.TEST_E2E_APP_URL  || 'http://localhost:3000/';
+const USERNAME = process.env.TEST_E2E_USERNAME || '';
```

Comment:
Hey @b5 , what should be the default user here if should be? If not then what should happen with the app when you try to test it without giving a user in configs.
Can you please explain what exactly should the script in package.json do?

Reply:
Hey hey! If you guys discussed this face to face, it'd be great to have a rundown here of what conclusions you guys came to.

Reply:
Yes, So basically I have added a check before the tests if the user is not specified - to log a message saying : 'Please provide credentials!'.